test/e2e/framework/helm.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2021 RadonDB.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	. "github.com/onsi/gomega"
+)
+
+func HelmInstallChart(release, ns string) {
+	args := []string{
+		"install", release, "./" + TestContext.ChartPath,
+		"--namespace", ns,
+		"--values", TestContext.ChartValues, "--wait",
+		"--kube-context", TestContext.KubeContext,
+		"--set", fmt.Sprintf("manager.tag=%s", TestContext.OperatorImageTag),
+	}
+
+	cmd := exec.Command("helm", args...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	Expect(cmd.Run()).Should(Succeed())
+}
+
+func HelmPurgeRelease(release, ns string) {
+	args := []string{
+		"delete", release,
+		"--namespace", ns,
+		"--kube-context", TestContext.KubeContext,
+	}
+	cmd := exec.Command("helm", args...)
```

Comment:
In Makefile， should check `helm` exists,   and install it

Reply:
I will add a document to prompt this.